tasks/model-comparison/main.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from coretex import currentTaskRun, Model
+
+
+def main() -> None:
+    taskRun = currentTaskRun()
+
+    models: list[Model] = taskRun.parameters["models"]
+    logging.info(">> [Model Comparison] Fetching models and starting comparison.")
```

Comment:
this log can be deleted, list of models is already fetched even before the main method is executed